sg_votes.cpp: unqualified call to 'std::move'

Rported by Clang 16:

```
Unvanquished/src/sgame/sg_votes.cpp:1055:24: warning: unqualified call to 'std::move' [-Wunqualified-std-cast-call]
        auto handler = [ vt = move( voteTemplate ), dt = move( displayTemplate ) ](
                              ^
                              std::
Unvanquished/src/sgame/sg_votes.cpp:1055:51: warning: unqualified call to 'std::move' [-Wunqualified-std-cast-call]
        auto handler = [ vt = move( voteTemplate ), dt = move( displayTemplate ) ](
                                                         ^
                                                         std::
```

--- a/src/sgame/sg_votes.cpp
+++ b/src/sgame/sg_votes.cpp
@@ -1052,7 +1052,7 @@ bool G_AddCustomVote( std::string vote, VoteDefinition def, std::string voteTemp
 		return false;
 	}
 	it = voteInfo.emplace( std::move( vote ), std::move( def ) ).first;
-	auto handler = [ vt = move( voteTemplate ), dt = move( displayTemplate ) ](
+	auto handler = [ vt = std::move( voteTemplate ), dt = std::move( displayTemplate ) ](
 					   gentity_t* ent, team_t team, std::string& cmd, std::string& arg,
 					   std::string& reason, std::string& name, int clientNum, int id )
 	{

